packages/turf-concave/bench.js

```diff
@@ -24,9 +24,8 @@ const fixtures = fs.readdirSync(directory).map(filename => {
  * pts3: 0.375ms
  */
 for (const {name, geojson} of fixtures) {
-    const {maxEdge, units} = geojson.properties || {maxEdge: 1};
     console.time(name);
-    concave(geojson, maxEdge, {units});
+    concave(geojson, geojson.properties);
```

Comment:
👍 I'm liking this Object `options`, you can simply give it the `geojson.properties` like you did, love it!

Reply:
Thanks Sir 😄